school_homework/OOP_project_game/settings.py

```diff
@@ -0,0 +1,19 @@
+""" Constants and settings used in the project """
+
+ALLOWED_ATTACKS = {'1': 'Paper',
```

Comment:
О, так у тебя есть уже допустимые атаки и моды, тогда тебе в тех местах где ты их используешь можно сделать новую функцию, типа get_allowed_options


```suggestion
def get_allowed_options(options_dict: dict) -> tuple:
    return tuple(option_dict.keys())
```
и просто подставлять нужный словарик)

Reply:
Так и сделал.